tests/unit/workflows/test_workflows_tasks_matching.py

```diff
@@ -89,115 +91,40 @@ def test_is_being_harvested_on_legacy_returns_true_when_there_is_one_core_catego
             },
         ],
     }
-    assert validate(record['arxiv_eprints'], subschema) is None
+    assert validate(obj.data['arxiv_eprints'], subschema) is None
+    assert belongs_to_relevant_category(obj, eng) is True
+    assert get_coreness(obj.data) is True
```

Comment:
You shoud be patching the ARXIV_CATEGORIES here, as you are relying on the default value.